src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -104,8 +106,34 @@ public static function getSubscribedEvents()
      *
      * @param \Exception $exception The \Exception instance
      * @param string     $message   The error message to log
+     *
+     * @deprecated since Symfony 4.4, use "logThrowable()" instead
      */
     protected function logException(\Exception $exception, $message)
+    {
+        @trigger_error(sprintf('The "%s::logException()" method is deprecated since Symfony 4.4, use "logThrowable()" instead.', __CLASS__), E_USER_DEPRECATED);
+
+        $this->doLogThrowable($exception, $message);
+    }
+
+    protected function logThrowable(\Throwable $exception, string $message): void
```

Comment:
There is no point for making this change. logThrowable is never called with anything but an Exception.

Reply:
I thought that once we catch \Throwable in HttpKernelnterface::handle() (which is the plan of https://github.com/symfony/symfony/pull/34232), we would catch \Throwable in this listener onKernelException method. Then, logThrowable() would be called with an \Error.

But @nicolas-grekas, you want \Error to pop here so I should revert right?

Reply:
I think it makes sense to widen the accepted type to prepare for the future where this will be needed.
What about duplicateRequest btw?

Reply:
Then it should be part of #34232. In this PR this change doesn't do anything valuable.

Reply:
removed